### ISSUE TITLE
when view user Channel page user activity missing link to user

### ISSUE THREAD

**Issue by spirogg** [MacWarrior/clipbucket-5.0#5]
```
> https://domain.com/user/admin
> User activity
> you see profile pic of user then to the right of it you see this:
> 
> <a href="/user/admin">admin</a> has uploaded a new photo
> 21 minutes ago
> 
> <a href="/user/admin">admin</a> has created a new collection
> 22 minutes ago
> 
> <a href="/user/admin">admin</a> has added a video to favorites
> 56 minutes ago
```
![user activity](https://user-images.githubusercontent.com/17736688/40581666-f1c7376e-6123-11e8-9fb9-44167200db18.PNG)

**Comment by MacWarrior** [comment]
Hi, 
How do you arrive to this url ? /view_user.php ?

Currently, all my users links redirect to /view_channel.php

I don't have SEO URLs activated but you see the difference ;)

**Comment by MacWarrior** [comment]
OK, I activated SEO URLs and now I got your page
But not the activity bloc

Have you activated something about it ?

**Comment by spirogg** [comment]
Maybe when I change SEO to different URL 

Seo video url pattern
 /video/ABCDXYZ/video-title
 /video-title-v123
 /video/123/video-title
 /video/123_video-title


![cb1](https://user-images.githubusercontent.com/17736688/40588199-6b36cbde-619f-11e8-8fda-bd27fd269cc9.PNG)
![cb2](https://user-images.githubusercontent.com/17736688/40588200-6b43354a-619f-11e8-8211-7b2ab674d08c.PNG)
![cb3](https://user-images.githubusercontent.com/17736688/40588201-6b4f8a52-619f-11e8-9c5a-5e873f4053e4.PNG)
![cb4](https://user-images.githubusercontent.com/17736688/40588202-6b5ae28a-619f-11e8-9461-16156bf1ac17.PNG)
![cb5](https://user-images.githubusercontent.com/17736688/40588203-6b6619d4-619f-11e8-8c9b-b6ef7f5b733a.PNG)

**Comment by spirogg** [comment]
dont have SEO on still issue updated to latest

**Comment by spirogg** [comment]
still have this issue ? set SEO OFF 
just fyi im just updating you 

thanks for fixing your version is SOLID :) and chromcast is awesome !!!!

**Comment by MacWarrior** [comment]
I didn't tell I fixed it yet :)

**Comment by spirogg** [comment]
hi I also notice in chrome browser I right click and inspect viewing console the date is not correct ?
see photo 

-
![date is not correct in user activity chrome console i see this](https://user-images.githubusercontent.com/17736688/40597279-1703279e-6206-11e8-91cb-60f1a2dac2f3.PNG)

-
![user activity time is not correct in console says 2016-03-31](https://user-images.githubusercontent.com/17736688/40597284-1f00b984-6206-11e8-9816-2ff3b2934ee2.PNG)

**Comment by spirogg** [comment]
also the video date is incorrect ?
![video shows date as 2016-15-03](https://user-images.githubusercontent.com/17736688/40597598-0aa392c0-6208-11e8-9102-add067cd9bc7.PNG)

**Comment by spirogg** [comment]
is this normal ? or is script set date instead of realtime - or server date ?

**Comment by spirogg** [comment]
Clipbucket script latest version from github same issue with date ?
![cb-script same thing](https://user-images.githubusercontent.com/17736688/40598015-d254c31e-620a-11e8-8087-03848d518aa3.PNG)

**Comment by MacWarrior** [comment]
Hi, 
This has been fixed in this commit : 334426c6921f206cd2d04ba0144809d46d6e1bfa
